app/assets/javascripts/discourse/lib/theme-selector.js.es6

```diff
@@ -12,21 +12,25 @@ export function currentThemeKey() {
   }
 }
 
-export function currentThemeId() {
-  let themeId = null;
-  let elem = _.first($(keySelector));
+export function currentThemeIds() {
+  const themeIds = [];
+  const elem = _.first($(keySelector));
   if (elem) {
-    themeId = elem.content;
-    if (_.isEmpty(themeId)) {
-      themeId = null;
-    } else {
-      themeId = parseInt(themeId);
-    }
+    elem.content
+      .split(",")
+      .map(num => parseInt(num, 10))
+      .filter(num => !isNaN(num))
+      .forEach(num => themeIds.push(num));
```

Comment:
I feel like we have too many loops here. Perhaps something like the following would work?

```
elem.content.split(",").forEach(num => {
  num = parseInt(num, 10);
  if (!isNaN(num)) { themeIds.push(num) };
})
```